unimol/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "unimol"
+version = "1.0.1"
+description = "Tools for Uni-Mol."
+license = "MIT"
+readme = "README.md"
+authors = [{ name = "DP Technology", email = "unimol@dp.tech" }]
+requires-python = ">=3.10,<4.0"
+dependencies = [
+    "numpy >=2.1.1",
+    "pandas >=2.2.3",
+    "scikit-learn-extra >=0.3.0",
```

Comment:
I think we will need to adjust these, mlbert is using `numpy==2.0.2`

Reply:
will do